clarity/src/vm/contexts.rs

```diff
@@ -1974,6 +1964,18 @@ impl CallStack {
     }
 }
 
+#[cfg(any(test, feature = "testing"))]
+impl<'a, 'hooks> OwnedEnvironment<'a, 'hooks> {
```

Comment:
Can you put this into the test module that uses it?  Thanks

Reply:
I'm not sure who to do that. If I move it into clarity/src/vm/tests/mod.rs, then I cannot access the private field, `self.context`.

Reply:
You can set `self.contract` to have `pub(crate)` visibility.